tilequeue/wof.py

```diff
@@ -892,6 +898,14 @@ def fetch_toi():
                 if n.wof_id in superseded_by_wof_ids:
                     diffs.append((n, None))
```

Comment:
Comparing this to the "funky" logic below, should be also be adding `n.wof_id` to `ids_to_delete` when it's superseded?